client/components/table-sort/table-sort.html

```diff
@@ -1,9 +1,7 @@
-<button type="button"
-        data-ng-click="sortableCtrl.sortElement()"
-        data-uib-tooltip="{{ this.title }}"
-        class="no-style no-space no-outline">
+<span data-ng-click="sortableCtrl.sortElement()"
```

Comment:
maybe keep a button element with "btn btn-link" class instead of clickable span ?